slick/src/sphinx/gettingstarted.rst

```diff
@@ -142,9 +142,9 @@ data:
 
 .. includecode:: code/FirstExample.scala#create
 
-The ``TableQuery``'s ``ddl`` method creates ``DDL`` (data definition language) objects
-with the database-specific code for creating and dropping tables and other
-database entities. Multiple ``DDL`` values can be combined with ``++`` to
+An implicit conversion of ``TableQuery`` gives us an object with driver-specific extension methods.
```

Comment:
Terminology has changed in 3.2. Replace "driver-specific" by "profile-specific".